README.md

```diff
@@ -102,13 +102,14 @@ Through [paging](https://www.syncfusion.com/blazor-components/blazor-datagrid/pa
+Today, we provide 1800+ controls and frameworks for web ([Blazor](https://www.syncfusion.com/blazor-components?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-datagrid-github-examples), [ASP.NET Core](https://www.syncfusion.com/aspnet-core-ui-controls?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-datagrid-github-examples), [ASP.NET MVC](https://www.syncfusion.com/aspnet-mvc-ui-controls?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-datagrid-github-examples), [ASP.NET WebForms](https://www.syncfusion.com/jquery/aspnet-webforms-ui-controls?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-datagrid-github-examples), [JavaScript](https://www.syncfusion.com/javascript-ui-controls?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-datagrid-github-examples), [Angular](https://www.syncfusion.com/angular-ui-components?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-datagrid-github-examples), [React](https://www.syncfusion.com/react-ui-components?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-datagrid-github-examples), [Vue](https://www.syncfusion.com/vue-ui-components?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-datagrid-github-examples), and [Flutter(Beta)](https://www.syncfusion.com/flutter-widgets?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-datagrid-github-examples)), mobile ([Xamarin](https://www.syncfusion.com/xamarin-ui-controls?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-datagrid-github-examples), [Flutter](https://www.syncfusion.com/flutter-widgets?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-datagrid-github-examples), [UWP](https://www.syncfusion.com/uwp-ui-controls?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-datagrid-github-examples), and [JavaScript](https://www.syncfusion.com/javascript-ui-controls?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-datagrid-github-examples)), and desktop development ([Windows Forms](https://www.syncfusion.com/winforms-ui-controls?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-datagrid-github-examples), [WPF](https://www.syncfusion.com/wpf-ui-controls?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-datagrid-github-examples), [WinUI(Preview)](https://www.syncfusion.com/winui-controls?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-datagrid-github-examples) and [UWP](https://www.syncfusion.com/uwp-ui-controls?utm_medium=listing&utm_source=github-examples&utm_campaign=blazor-datagrid-github-examples)). We provide ready-to-deploy enterprise software for dashboards, reports, data integration, and big data processing. Many customers have saved millions in licensing fees by deploying our software.
```

Comment:
Remove beta and include MAUI in the ,ist
<img width="143" alt="image" src="https://github.com/SyncfusionExamples/blazor-grid-getting-started/assets/61148953/73d4ea30-f7e8-44c9-974d-63ede92c0d89">

Reply:
Resolved